src/meta/test/JobManagerTest.cpp

```diff
@@ -711,11 +709,11 @@ TEST_F(JobManagerTest, RecoverJob) {
                                          jd.getErrorCode());
       jobMgr->save(jobKey, jobVal);
     }
-    auto nJobRecovered = jobMgr->recoverJob(spaceId, nullptr, {1});
+    auto nJobRecovered = jobMgr->recoverJob(spaceId, nullptr, {21});
     ASSERT_EQ(nebula::value(nJobRecovered), 0);
 
     nJobRecovered = jobMgr->recoverJob(spaceId, nullptr);
-    ASSERT_EQ(nebula::value(nJobRecovered), 2);
```

Comment:
The result here was wrong before

Reply:
it should be 2, 1 FLUSH and 1 ZONE_BALANCE

Reply:
just 1
The ZONE_BALANCE of the latest jobId 28 is finished, so the ZONE_BALANCE of the job id less than 28 does not need to be recovered.

Reply:
please don't change the test's logic

Reply:
This is what I achieved at the time.

Reply:
It's amazing that the test case can be passed before?

Reply:
`The ZONE_BALANCE of the latest jobId 28 is finished, so the ZONE_BALANCE of the job id less than 28 does not need to be recovered`
This case was originally added by me and later changed by you. please correct

Reply:
the test was right in the past,  you changed the logic of test.
I created a new JobManager in each segment, so the val should be 2, it should recover 1 FLUSH and 1 ZONE_BALANCE. 
please just make sure the test could cover recovering both balance type and un-balance type.

Reply:
ok my mistake

Reply:
why don't you create a new jobmanager in each segment

Reply:
ok deque in each segment is also ok